modules/ppcp-paypal-subscriptions/src/PayPalSubscriptionsModule.php

```diff
@@ -660,6 +693,11 @@ private function update_subscription_product_meta( WC_Product $product, Subscrip
 			if ( $product->meta_exists( 'ppcp_subscription_product' ) && $product->meta_exists( 'ppcp_subscription_plan' ) ) {
 				$subscriptions_api_handler->update_product( $product );
 				$subscriptions_api_handler->update_plan( $product );
+
+				if ( $product->get_meta( '_ppcp_enable_subscription_product', true ) === 'yes' ) {
+					update_metadata( 'post', $product->get_id(), '_sold_individually', 'yes', 'no' );
```

Comment:
Better use WC Product API to [set sold individually](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/includes/abstracts/abstract-wc-product.php#L1013-L1015):

![Captura de pantalla 2024-05-29 a las 11 28 10](https://github.com/woocommerce/woocommerce-paypal-payments/assets/456223/cc51b0fa-4e8c-4525-9cc7-cc1ba1e1d6bb)